src/w3c/style.js

```diff
@@ -10,15 +10,15 @@ import { pub, sub } from "core/pubsubhub";
 export const name = "w3c/style";
 function attachFixupScript(doc, version) {
   const script = doc.createElement("script");
-  script.addEventListener(
-    "load",
-    function() {
-      if (window.location.hash) {
+  if (location.hash) {
+    script.addEventListener(
+      "load",
+      () => {
         window.location = window.location;
```

Comment:
What does this line do?

Reply:
>What does this line do? 

forces page to jump to hash after page load.

Not sure if this `eventListener` is needed. If a URL has hash, browser itself should trigger the jump.

Reply:
Unfortunately, this one is needed because fixup.js moves stuff around after ReSpec finishes (screwing up the scroll position). This should just set the hash tho.